### DISCUSSION TITLE
Mozilla support page causes SingleFile to display "This page cannot be saved with SingleFile"

### DISCUSSION THREAD

**Discussion by SteveS73** [gildas-lormeau/SingleFile#717] [Q&A] [answered]
**Describe the bug**
When I tried to save the following Mozilla Support page, the SingleFile icon displays a "stop" icon over the top of itself. Hovering the mouse over the icon displays "This page cannot be saved with SingleFile."  https://support.mozilla.org/en-US/kb/remove-other-bookmarks-folder-from-bookmarks-toolbar#firefox:win10:fx84

**To Reproduce**
1. Go to https://support.mozilla.org/en-US/kb/remove-other-bookmarks-folder-from-bookmarks-toolbar#firefox:win10:fx84
2. Click on SingleFile add-on icon

**Expected behavior**
In the past I've always been able to save such support pages. Maybe it's due to a change by Mozilla?

**Environment**
 - Win10 Pro 1909 64bit
 - Browser:  Firefox
 - Version: 84.0.2 64 bit

**Comment by gildas-lormeau** [comment, marked as answer]
This is a [known issue](https://github.com/gildas-lormeau/SingleFile#known-issues). Browser providers block content scripts of extensions on some domains they own. This can be fixed by removing `support.mozilla.org` from `extensions.webextensions.restrictedDomains` in the `about:config` page.

**Reply by zerobaka** [reply]
No use, even when i cleared the whole text from extensions.webextensions.restrictedDomains, after restarting firefox, singlefile still refused to save any pages started with this address "https://addons.mozilla.org/en-US/firefox/addon/". Any other tips, please?

**Comment by SteveS73** [comment]
Worked, Thanks!

**Comment by bansan85** [comment]
Just to found more easily this page, please found below the default list of restricted url in Firefox:

```
accounts-static.cdn.mozilla.net
accounts.firefox.com
addons.cdn.mozilla.net
addons.mozilla.org
api.accounts.firefox.com
content.cdn.mozilla.net
discovery.addons.mozilla.org
install.mozilla.org
oauth.accounts.firefox.com
profile.accounts.firefox.com
support.mozilla.org
sync.services.mozilla.com
```

**Reply by gildas-lormeau** [reply]
Thank you for the additional details. I converted the issue into a discussion thread to make it more visible, hopefully.

**Reply by vlgmod** [reply]
Source: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#restricted_domains

**Comment by iG8R** [comment]
Hello.
Is there some way to do the same but in Edge?
PS. I have plenty mht files and only Edge can open them so I want to save them with SingleFile.

**Reply by gildas-lormeau** [reply]
Sorry for the late reply. As far as I know, you cannot do the same in Chromium-based browsers.

**Reply by J-HS-L** [reply]
@gildas-lormeau 

No way, is this still definitely the case?!?
![Screenshot 2025-06-25 234550](https://github.com/user-attachments/assets/ae7f457c-057d-47a9-843f-e629160cedbb)
 
I use edge as my primary browser because; 
I like its combo of - features + (ok-ish) perf/stability/reliability!

is there some way I can circumvent via ssheets or uscrpts & uB0?
I want to use singlefile to save my most useful interactions!

**Reply by gildas-lormeau** [reply]
@J-HS-L What pages are you trying to save?

**Reply by J-HS-L** [reply]
@gildas-lormeau 

Copilot website

**Reply by J-HS-L** [reply]
@gildas-lormeau  ?

see here
https://copilot.microsoft.com/chats/VgHK6KSD3hUFKoiX6msQ6

**Reply by vlgmod** [reply]
I successfully download with Firefox and Chrome.

[Downloads.zip](https://github.com/user-attachments/files/20977560/Downloads.zip)

**Reply by J-HS-L** [reply]
so should be doable edge?

**Reply by vlgmod** [reply]
You're right, you can't do it with Edge.

I read too quickly "Chromium-based browsers" so I checked on the first web navigator I have.

**Reply by J-HS-L** [reply]
@vlgmod Why can we do it with chrome & ff, but not edge??
Presumably you actually tried Edge? I guess my only option is to switch to my FF env?!

**Reply by vlgmod** [reply]
I have no idea. I just tested as a user but I'm not a web dev. Yes, I tried with Edge and current workaround is to use Firefox.
